utils/code_gen/codegen_components.py

```diff
@@ -154,30 +158,31 @@ def route_question_to_code(self, state):
             print("---ROUTE QUESTION TO CODEGEN---")
             return "codegen"
         
-    def pass_to_codegen(self, state: Dict) -> Dict:
+    def pass_to_codegen(self, state: dict) -> dict:
         """
         Get a new query based on the given state.
 
         Args:
-            state (Dict): A dictionary containing the current state.
+            state: A dictionary containing the current state.
 
         Returns:
-            Dict: The state updated dictionary containing the new query.
+            The state updated dictionary containing the new query.
         """
 
         question: str = state["original_question"]
 
         return {"original_question": question}
         
-    def code_generation(self, state: Dict) -> Dict:
+    def code_generation(self, state: dict) -> dict:
         """
         Generates code based on the given question.
 
         Args:
-            state (Dict[str, str]): The state dictionary containing the question to generate code for.
+            state: The state dictionary containing the question 
+            to generate code for.
```

Comment:
Why did you interrupt this line, and others similar lines?

Reply:
Interrupt?  Do you mean break up the line into new lines?  If so, length was deemed too long by Ruff, so it was for style.